tests/testthat/setup.R

```diff
@@ -1,5 +1,6 @@
 # load common required test packages
 library(ggplot2, quietly = TRUE)
+library(data.table)
```

Comment:
the only danger with doing this is if you have internally not exported data.table functions then this will allow that not to be caught.